force-app/main/default/lwc/importContactsFromDevice/importContactsFromDevice.html

```diff
@@ -0,0 +1,39 @@
+<template>
+    <lightning-card title="Contacts" icon-name="standard:location">
```

Comment:
Open question: do we want to simply list contacts from the device or really import them as the name of the component implies?
I'm fine with just listing them but in that case we should rename the component (folder, files and JS class). Also, we'll want the card title to match the recipe name.